sympy/physics/mechanics/tests/test_jointsmethod.py

```diff
@@ -0,0 +1,82 @@
+from sympy import symbols, Matrix, cos, sin, expand
+from sympy.physics.mechanics import (PinJoint, JointsMethod, Body, KanesMethod,
+                                    PrismaticJoint, LagrangesMethod)
+from sympy.physics.vector import dynamicsymbols
+
+
+t = dynamicsymbols._t
+
+
+def test_jointsmethod():
+    P = Body('P')
+    C = Body('C')
+    Pin = PinJoint('P1', P, C)
+    C_ixx, g = symbols('C_ixx g')
+    theta, omega = dynamicsymbols('theta_P1, omega_P1')
+    P.apply_force(g*P.y)
+    method = JointsMethod(P, Pin)
+    assert method.frame == P.frame
+    assert method.bodylist == [C, P]
+    assert method.forcelist == [(P.masscenter, g*P.frame.y)]
```

Comment:
This does nothing in this problem.

Reply:
But it's a nice check that all bodies are being considered in forcelist